app/controllers/application_controller.rb

```diff
@@ -1,20 +1,24 @@
 class ApplicationController < ActionController::Base
+   
+    rescue_from CanCan::AccessDenied, with: :unauthorized_access
```

Comment:
if key hash is last parameter dont need {} braces, if not do need {} braces, best practice to keep hash as last argument/parameter